dhcpv4/client.go

```diff
@@ -165,18 +221,99 @@ func (c *Client) Exchange(ifname string, discover *DHCPv4, modifiers ...Modifier
 	conversation = append(conversation, request)
 
 	// Ack
-	ack, err := BroadcastSendReceive(sfd, rfd, request, c.ReadTimeout, c.WriteTimeout, MessageTypeAck)
+	ack, err := c.sendReceive(sfd, rfd, request, MessageTypeAck)
 	if err != nil {
 		return conversation, err
 	}
 	conversation = append(conversation, ack)
 	return conversation, nil
 }
 
+// SendReceive sends a packet (with some write timeout) and waits for a
```

Comment:
the `S` is now lower-case
```suggestion
// sendReceive sends a packet (with some write timeout) and waits for a
```